apps/emqx/test/emqx_persistent_session_ds_state_fuzzer.erl

```diff
@@ -356,14 +351,6 @@ sample(Size) ->
 precondition(_, _) ->
     true.
 
-postcondition(S, {call, ?MODULE, iterate_sessions, [_]}, Result) ->
```

Comment:
I removed this check for simplicity: listing the sessions is a dirty operation, so it tends to create unnecessary problems for the test. Iteration is currently done using a common DS API that has its own tests, so there's no need to test it here.